geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartGatewaySenderCommand.java

```diff
@@ -155,4 +160,38 @@ public ResultModel startGatewaySender(@CliOption(key = CliStrings.START_GATEWAYS
 
     return resultModel;
   }
+
+  private boolean sendAllStartingMessage(Set<DistributedMember> dsMembers, Cache cache,
```

Comment:
I wonder why you are doing it through JMX, have you tried having a function to turn on/off this flag and just execute that function on the other members? That's a common way to tell members to do something

Reply:
Do you have an example from other part of the code where it is done as you are suggesting?
Do you mean executing the function once for each member or having a function that executes on all members (the latter would seem more convenient)?

Reply:
You can check here for an example: https://github.com/apache/geode/blob/eb8668ad7caa6e20d9be242ad719e173ebf244d8/geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyRegionCommand.java#L77

Reply:
I have added a new commit with a change to function invocation instead of doing it through JMX.